mesheryctl/pkg/utils/error.go

```diff
@@ -227,7 +227,7 @@ func ErrAttachAuthToken(err error) error {
 
 func ErrFailRequest(err error) error {
 	return errors.New(ErrFailRequestCode, errors.Alert, []string{"Failed to make a request"},
-		[]string{err.Error()}, []string{}, []string{})
+		[]string{"Connection refused. Run `mesheryctl system check` to verify that your environment is ready."}, []string{}, []string{})
```

Comment:
Good deal. This is an improvement. Ideally, we'll have the probable cause and suggested remediations filled in here, too.

Reply:
@leecalcote yes, I'll find a message to put in there too.
Do you think it would be worth to put somewhere a log.Debug the plain error? So that by running the command in verbose we can see clearly the endpoint and other useful information.
Or we can pass it along with a message in the probable cause field

Reply:
@abdullah1308 ...

Reply:
@suhail34 ...